ui/desktop/tests/acceptance/projects/targets/index-test.js

```diff
@@ -106,7 +106,7 @@ module('Acceptance | projects | targets | index', function (hooks) {
       type: 'org',
       scope: globalScope,
     });
-    const org2Scope = { id: instances.scopes.org2.id, type: 'org' };
+    const org2Scope = { id: instances.scopes.org.id, type: 'org' };
```

Comment:
non-blocking: I am curious why this was changed? If it does need to be changed, could we remove this variable and use `orgScope` from line 100?

Reply:
I am good with that approach. Thanks for explaining.